raml/APIs/schemas/certs_response.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Certificate Response",
+  "description": "Array of certificates to validate Access Token",
```

Comment:
During the workshop we figured out that the following format would be preferable on the `/certs` endpoint:
```
{"default": "<cert_string_here>"}
```
This would keep the option open for a certificate exchange / renewal if desired later on.

Reply:
Hi Jasper, sorry for the delay. I think we are leaning more towards the idea of an array type now at the /certs endpoint, and implementing a method in which tokens are attempted to be validated against each certificate in the array. Its unlikely there will ever be more than a handful of certificates at this endpoint, and this way it prevents the need for keywords / specific indexing.